main.js

```diff
@@ -844,21 +844,23 @@ async function createWindow() {
         newValue: settingsAccelerator['miniplayer-open-close'],
     })
 
-    globalShortcut.register('MediaPlayPause', () => {
-        checkDoubleTapPlayPause()
-    })
+    if (!settingsProvider.get('settings-windows10-media-service-show-info')) {
+        globalShortcut.register('MediaPlayPause', () => {
+            checkDoubleTapPlayPause()
```

Comment:
Add a guradner here instead of commenting out the register statements.

Or alternatively watch the changes of `settings-windows10-media-service-show-info` and do it in register/unregister manner.

Reply:
These settings require an app restart whenever changed. I figured this is not necessary

Reply:
Cool! Sorry for missing that. Thanks for your contribution!